### ISSUE TITLE
Lacking task `copyFrameworkResourcesToApp` in 0.24.0-alpha-4

### ISSUE THREAD

**Issue by kevincianfarini** [icerockdev/moko-resources#646]
Hi. I’m working on upgrading our non-trivial app from moko-resources 0.23.0 > 0.24.0-alpha-4. I’ve nearly worked through all of the breaking changes, but I’m seeing the following error when trying to link our iOS framework.

```
Cannot locate tasks that match ':shared:octopus:copyFrameworkResourcesToApp' as task 'copyFrameworkResourcesToApp' not found in project ':shared:octopus'.
```

I noticed in the multiple alpha releases this has come up as a bug that’s needed resolution. When I check what tasks are available on our umbrella shared framework, I see the following:

```
Moko-resources tasks
--------------------
copyResourcesDebugFrameworkIosSimulatorArm64
copyResourcesDebugTestIosSimulatorArm64
copyResourcesReleaseFrameworkIosSimulatorArm64
```

We are setting the base name of our prefix in our framework.
```
        binaries {
            framework {
                baseName = "shared"
                isStatic = true
```

Does this sound like a bug with moko-resources, or is there something obvious that we’re doing wrong?

**Comment by stakancheck** [comment]
Same problem with static framework on alpha 4.

**Comment by kevincianfarini** [comment]
Glad it not just us!

**Comment by Alex009** [comment]
should be fixed in 0.24.0-alpha-5 (now deploy in progress)

**Comment by ExNDY** [comment]
Hi, can u share configuration of xcode task in your project?
